src/app/feature/consultant/services/consultant-store/consultant-store.service.ts

```diff
@@ -14,7 +14,7 @@ export const blankConsultant: Consultant = {
   firstName: 'First',
   lastName: 'Last',
   title: 'Title',
-  practice: 'Practice',
+  practice: '',
```

Comment:
I see on your branch that this prevents the string `Practice` from being sent to the DB when a new consultant is being added because the required validation fails on an empty string. Why not set to `null` though the way `status` is set to `null`, for consistency?

Also, wouldn't the same need to be done for `firstName`, `lastName` 'title`, and `email` to keep these default values from being added to the DB? I see that setting `phone` to `null` causes the pipeline to break bc of Natalie's comment. I just picked up ticket 83 where I will fix the `phone` field.

One last comment, the string `Practice` no longer gets displayed next to `Title` as a placeholder (see screenshot). I don't know much about Angular but I think the issue here is about distinguishing *default* values from *placeholder* values. For my ticket 83 I'll be looking into this as well ...
![screenshot](https://user-images.githubusercontent.com/47640182/64133275-95006700-cd9a-11e9-95d0-fc1efa7b31d4.png)